README.md

```diff
@@ -529,11 +529,14 @@ This repository contains a curated list of awesome open source libraries that wi
 
 
 ## Commercial Platform
-* [Amazon SageMaker](https://aws.amazon.com/sagemaker/) - End-to-end machine learning development and deployment interface where you are able to build notebooks that use EC2 instances as backend, and then can host models exposed on an API.
+* [Amazon Web Services](https://aws.amazon.com/) - Amazon Web Services offers reliable, scalable, and inexpensive cloud computing services to productionalize machine learning, including: [Amazon Augmented AI](https://aws.amazon.com/augmented-ai/), [Amazon Rekognition](https://aws.amazon.com/rekognition/), [Amazon SageMaker](https://aws.amazon.com/sagemaker/).
```

Comment:
This looks better, I would remove inexpensive as that's not something I've heard describe organisations running SageMaker at scale (if anything, the opposite, but other than that looks good

Reply:
Done